og_access/og_access.install

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Drupal\og\Og;
+
+/**
+ * @file
+ * Install/update/uninstall hook implementations.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function og_access_install() {
+  /** @var \Drupal\og\GroupTypeManager $group_type_manager */
+  $group_type_manager = \Drupal::service('og.group_type_manager');
+  $groups = $group_type_manager->getAllGroupBundles('node');
+
+  foreach ($groups as $group) {
+    Og::createField(OG_ACCESS_FIELD, 'node', $group);
```

Comment:
are we doing this in OG7 as-well? Seems like we don't have to do it on install, and maybe have some UI to do it for us?